livestyled/resource_client.py

```diff
@@ -577,12 +577,12 @@ def get_push_consent(
 
     def get_tickets(
             self,
-            external_ticket_id: str or None = None,
+            external_ticket_ids: dict or None = None,
             user: User or str or int or None = None,
     ) -> Generator[Ticket, None, None]:
         filters = {}
-        if external_ticket_id:
-            filters['externalTicketId'] = external_ticket_id
+        if external_ticket_ids:
+            filters['externalTicketId[]'] = external_ticket_ids
```

Comment:
In SecuTix integration `externalTicketId` might either be set using movementId or ticketId. So when we're syncing tickets we need to search by both of these values to prevent duplicates.

On API V4 side it happily accepts array for `externalTicketId` filter so I'm just making python-sdk compatible with API V4.

Of course I could just make a couple of separate requests when syncing tickets, but thought if API V4 already supports searching by array for this field, then why making a couple of requests when I can achieve the same result with just a single request.